src/runtime-rs/crates/runtimes/common/src/message.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use anyhow::Result;
+use tokio::sync::mpsc::{channel, Receiver, Sender};
+
+#[derive(Debug)]
+pub enum Action {
+    StartListen,
+    StopListen,
+    Shutdown,
+}
+
+#[derive(Debug)]
+pub struct Message {
+    pub action: Action,
+    pub resp_sender: Option<Sender<Result<()>>>,
+}
+
+impl Message {
+    pub fn new(action: Action) -> Self {
+        Message {
+            action,
+            resp_sender: None,
+        }
+    }
+
+    pub fn new_with_receiver(action: Action) -> (Receiver<Result<()>>, Self) {
+        let (resp_sender, receiver) = channel(100);
```

Comment:
Constant `100` is a bit arbitrary here (and copy-pasted from examples). Was there any evaluation done about the buffer size we need?

Suggestion: Add a constant for that buffer size, and a comment explaining the value.

Reply:
Thanks for your review, already use constant and explain the value.